#! /bin/sh -e

## DP: fix the shebangs in the utils
## DP: Author: Fabio Tranchitella

. $(dirname $0)/DPATCH

exit 0
@DPATCH@
diff -urN postgis-1.3.5/utils/create_undef.pl postgis-1.3.5.debian/utils/create_undef.pl
--- postgis-1.3.5/utils/create_undef.pl	2005-07-13 16:26:47.000000000 +0200
+++ postgis-1.3.5.debian/utils/create_undef.pl	2009-03-05 23:03:54.000000000 +0100
@@ -1,3 +1,5 @@
+#!/bin/sh
+
 eval "exec perl -w $0 $@"
 	if (0);
 
diff -urN postgis-1.3.5/utils/postgis_proc_upgrade.pl postgis-1.3.5.debian/utils/postgis_proc_upgrade.pl
--- postgis-1.3.5/utils/postgis_proc_upgrade.pl	2005-12-16 09:33:04.000000000 +0100
+++ postgis-1.3.5.debian/utils/postgis_proc_upgrade.pl	2009-03-05 23:03:38.000000000 +0100
@@ -1,4 +1,4 @@
-#/bin/sh
+#!/bin/sh
 
 #
 # This script produces an .sql file containing
diff -urN postgis-1.3.5/utils/postgis_restore.pl postgis-1.3.5.debian/utils/postgis_restore.pl
--- postgis-1.3.5/utils/postgis_restore.pl	2008-10-02 17:59:41.000000000 +0200
+++ postgis-1.3.5.debian/utils/postgis_restore.pl	2009-03-05 23:03:29.000000000 +0100
@@ -1,4 +1,4 @@
-#/bin/sh
+#!/bin/sh
 #
 # This script is aimed at restoring postgis data
 # from a dumpfile produced by pg_dump -Fc
